webpack_config/electron.production.js

```diff
@@ -20,7 +20,7 @@ const main = merge.smart(electronMain, {
 });
 
 const render = merge.smart(electronRender, {
-  target: 'electron-renderer',
+  target: 'web',
```

Comment:
Can we please add a comment as documentation for our future selfs.
It's important to know that this is related to `nodeIntegration`

Reply:
Thanks!